cloud-function/src/handlers/render-html.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable n/no-unsupported-features/node-builtins */
+import type { NextFunction, Request, Response } from "express";
+import type { IncomingMessage, ServerResponse } from "node:http";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { renderHTML } from "../internal/ssr/main.js";
+import { sourceUri, Source } from "../env.js";
+import { withRenderedContentResponseHeaders } from "../headers.js";
+
+const target = sourceUri(Source.content);
+
+export async function handleRenderHTML(
```

Comment:
For consistency with the other handlers, I would suggest:

```suggestion
export async function renderIndexHTML(
```